src/odemis/acq/stream/_projection.py

```diff
@@ -1014,9 +1014,15 @@ def _projectTile(self, tile):
         dims = tile.metadata.get(model.MD_DIMS, "CTZYX"[-tile.ndim::])
         ci = dims.find("C")  # -1 if not found
         # is RGB
+        tint = tuple(self.stream.tint.value)
```

Comment:
I assume that tint converts a stream from grayscale to (for example) a tinted "blue"scale? Is that right?
Would be nice to add somewhere a comment which more explicitly describes the function to tint

Reply:
Indeed, I forgot to update the comments in this function. Now done.